src/styles.css

```diff
@@ -46,3 +81,11 @@
   -moz-osx-font-smoothing: grayscale;
   font-size: 1rem;
 }
+
+.hill-chart-light .hill-chart-group > text {
+  fill: #000;
+}
+
+.hill-chart-group > text {
```

Comment:
Codacy found an issue: [Expected selector ".hill-chart-group > text" to come before selector ".hill-chart-light .hill-chart-group > text" (no-descending-specificity)](https://app.codacy.com/manual/nagi1/hill-chart/pullRequest?prid=5736858)